core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -37,6 +37,7 @@ oppia.controller('FeedbackTab', [
       oppiaDatetimeFormatter.getLocaleAbbreviatedDatetimeString);
 
     $scope.activeThread = null;
+    $scope.allowResponseToFeedback = GLOBALS.userIsLoggedIn;
```

Comment:
I think we should stick with "userIsLoggedIn" only, as it is used in various other files ([here](https://github.com/oppia/oppia/search?utf8=%E2%9C%93&q=+%24scope.userIsLoggedIn&type= )) and it's better to maintain consistency between files.

Reply:
ok will change that.

Reply:
I have changed the variable name.